src/mame/layout/tek4404.lay

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<!--
+license:CC0-1.0
+authors: Elektraglide
+
+Tektronix 4404 Artificial Intelligence System.
+-->
+<mamelayout version="2">
+	<element name="static_black"><rect><color red="0.0" green="0.0" blue="0.0" /></rect></element>
+	<element name="led_round" defstate="1">
+		<disk state="1">
+			<color red="0.2" green="0.0" blue="0.0" />
+		</disk>
+		<disk state="0">
+			<color red="1.0" green="0.0" blue="0.0" />
+		</disk>
+	</element>
+	<element name="led_rect" defstate="1">
+		<rect state="1">
+			<color red="0.2" green="0.0" blue="0.0" />
+		</rect>
+		<rect state="0">
+			<color red="1.0" green="0.1" blue="0.1" />
+		</rect>
+	</element>
+	<view name="Graphics">
+		<bounds x="0" y="0" width="640" height="496" />
+		
+		<screen index="0">
+			<bounds x="0" y="0" width="640" height="480" />
+		</screen>
+		<element name="spacer" ref="static_black">
+			<bounds x="0" y="480" width="640" height="16" />
+		</element>
+
+		<element name="led_1" ref="led_round">
+			<bounds x="46" y="484" width="8" height="8" />
+		</element>
+		<element name="led_2" ref="led_round">
+			<bounds x="34" y="484" width="8" height="8" />
+		</element>
+		<element name="led_4" ref="led_round">
+			<bounds x="22" y="484" width="8" height="8" />
+		</element>
+		<element name="led_8" ref="led_round">
+			<bounds x="10" y="484" width="8" height="8" />
+		</element>
+		<element name="led_disk" ref="led_rect">
+			<bounds x="614" y="485" width="16" height="6" />
+		</element>
```

Comment:
Would it be a good idea to add text labels for the LEDs?  As it is, there are three identical-looking round LEDs, the function of which won’t be clear without examining the source.  Also, the outputs corresponding to these don’t appear to actually be hooked up?